inc/checks/class-language-update.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+
+/**
+ * Warns when there are language updates available.
+ */
+class Language_Update extends Check {
+
+	public function run() {
+		ob_start();
+		WP_CLI::run_command( array( 'language', 'core', 'list' ), array( 'format' => 'json' ) );
```

Comment:
Use `WP_CLI::runcommand()` instead of `WP_CLI::run_command()` (note the underscore).

The former can return and parse the result, simplifying the code:

```php
$languages = WP_CLI::runcommand(
   'language core list --format=json',
   array(
      'return' => true,
      'parse'  => 'json',
      'launch' => false,
   )
);
```